spec/unit/build_audit_trigger_spec.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+require_relative '../../lib/audit_tables/build_audit_trigger'
+
+describe AuditTables::BuildAuditTrigger do
+  it 'can be' do
+    expect(described_class.new({})).to be_an_instance_of(AuditTables::BuildAuditTrigger)
+  end
+end
```

Comment:
This are basically 'compiler' tests - they were failing until the proper requires were added, duplicating what was occurring on staging.